src/libraries/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -961,7 +1012,7 @@ public sealed partial class Region : System.MarshalByRefObject, System.IDisposab
         public bool IsVisible(System.Drawing.Rectangle rect, System.Drawing.Graphics? g) { throw null; }
         public bool IsVisible(System.Drawing.RectangleF rect) { throw null; }
         public bool IsVisible(System.Drawing.RectangleF rect, System.Drawing.Graphics? g) { throw null; }
-        public bool IsVisible(int x, int y, System.Drawing.Graphics g) { throw null; }
+        public bool IsVisible(int x, int y, System.Drawing.Graphics? g) { throw null; }
```

Comment:
Intentional change?

Reply:
This was from running GenerateReferenceSource it seems like we missed this nullable annotation in the ref.